tests/simulator/test_restarts.py

```diff
@@ -295,22 +292,23 @@ def test_restarts_elapsed_time(self, ndim, interp, simInput, expected_num_levels
         time_step = simput["time_step"]
         time_step_nbr = simput["time_step_nbr"]
 
-        restart_idx = 4
+        restart_idx = 1
         restart_time = time_step * restart_idx
-        timestamps = [time_step * restart_idx, time_step * time_step_nbr]
+        timestamps = [time_step * time_step_nbr]
 
         # first simulation
         local_out = self.unique_diag_dir_for_test_case(
             f"{out}/elapsed_test", ndim, interp
         )
-        simput["restart_options"]["dir"] = local_out
-        import datetime
+        diag_dir0 = local_out
+        diag_dir1 = f"{local_out}_n2"
 
-        seconds = 9
+        seconds=1 # dump on first advance always!
```

Comment:
how are we sure what the comment says is true?

Reply:
with `autodump=false` I think it's correct